src/terminal.rs

```diff
@@ -498,6 +537,132 @@ fn compute_inline_size<B: Backend>(
     ))
 }
 
+/// A consistent view into the terminal state for rendering a single fram.
```

Comment:
```suggestion
/// A consistent view into the terminal state for rendering a single frame.
```

Reply:
fixed in https://github.com/ratatui-org/ratatui/commit/0d6bb1073447c18ac0a9be4f7319ccb1f3d4968b